### DISCUSSION TITLE
ant-design switch

### DISCUSSION THREAD

**Discussion by Liu20011018** [ant-design/ant-design#52044] [General]
When using the Switch component, there will be errors in the image.
![Snipaste_2024-12-18_16-31-21](https://github.com/user-attachments/assets/923ec789-cbab-4384-950e-b6b62aa49563)

**Comment by petercat-assistant** [comment]
It seems like you're experiencing an issue with the Switch component in Ant Design, where there are errors related to an image. To assist you better, could you please provide more details about the error message or the context in which this issue occurs? This information will help in diagnosing the problem more effectively.

If you have any code snippets or additional screenshots, please share them as well.

**Reply by Liu20011018** [reply]
Thank you very mush, this issue has already been resolved. It become normal when I lowered the React version back to 18.

**Comment by zombieJ** [comment]
antd not support React 19 yet. Ref https://github.com/ant-design/ant-design/discussions/51919

**Reply by Liu20011018** [reply]
Thank you very mush，it's just this issue. It become normal when I lowered the React version back to 18.
